doc/radosgw/frontends.rst

```diff
@@ -59,6 +59,17 @@ Options
 :Type: String
 :Default: None
 
+``tcp_nodelay``
+
+:Description: If set the socket option will disable Nagle's algorithm on 
+              the connection which means that packets will be sent as soon 
+              as possible instead of waiting for a full buffer or timeout to occur.
+              ``1`` will Disable Nagel's algorithm for all sockets.
+              ``any other value`` will Keep the default: Nagel's algorithm enabled
```

Comment:
these last couple sentences are rendered without line breaks, so the capitalization looks awkward:
> ... timeout to occur. 1 will Disable Nagel’s algorithm for all sockets. any other value will Keep the default: Nagel’s algorithm enabled

Reply:
Actually it is the same as the civetweb description for tcp_nodelay

Reply:
What do you think about to keep it as is here: 
https://github.com/civetweb/civetweb/blob/master/docs/UserManual.md#tcp_nodelay-0

Reply:
Changed it